### ISSUE TITLE
Server showing as version 0.218.12

### ISSUE THREAD

**Issue by Sumokai31** [lloesche/valheim-server-docker#673]
Hi everybody,

I've some issues concerning my valheim container.
I'm running a default compose file, very simple:

> services: 
>   valheim-server:
>     container_name: valheim-server
>     image: ghcr.io/lloesche/valheim-server
>     cap_add:
>       - sys_nice
>     volumes: 
>       - /opt/docker/data/valheim-server/config:/config
>       - /opt/docker/data/valheim-server/data:/opt/valheim
>     ports: 
>       - "2456-2458:2456-2458/udp"
>       - "8098:9001/tcp"
>     environment:
>       - PUID=1000
>       - PGID=1000
>       - TZ= Europe\Paris
>       - SERVER_NAME=Server
>       - WORLD_NAME=World
>       - SERVER_PASS=StrongPassword!
>       - SERVER_PUBLIC=false
>       - BACKUPS_MAX_AGE=30
>       - PUBLIC_TEST=false
>     restart: unless-stopped

My issue is that in docker logs I've a game version listed as 0.217.48 which is what I want but when I try to connect to the server from my computer, it's showing version 0.218.12 for the server and tell me that I'm running an out of date game.
I don't know also why the server name is THUNDERDOME, I've never set that.

![image](https://github.com/lloesche/valheim-server-docker/assets/78795023/a59c0d36-04f9-4534-9bf6-f309409c0d6d)

Have you some ideas?

**Comment by Sumokai31** [comment]
Some aditionnals information for my case:
First before last docker image update, I've manage to connect to this server without issue.
I've turned it off and just turned it on yesterday for playing on it so I don't think there is something to do with my configuration.

**I've tried several things:**

- The name **THUNDERDOME** only pops when I put the <server ip>:2456 and not 2457 which is a mistake on my side :-).
- I can't connect to my server throught the game menu with <server ip>:2457, I've a "**failed to connect**" error but if I add the server throught Steam servers with the port 2457, my server appears in the list with the right name and map but not with the right port on the  IP Address section (I put a picture of that below). But if I try to connect on it I can even get to the password server question but after it failed with an "**Incompatible version**".

![image](https://github.com/lloesche/valheim-server-docker/assets/78795023/347b1240-93df-40e3-86c8-a8b299fcb68b)

- I've tried with an other laptop to connect on the server, no success too.

Is there a way to specified the valheim server version we want on the server?

**Comment by Sumokai31** [comment]
Some news from my case.

It's nothing to do regarding the container, even with the container off my IP 192.168.1..10:2456 continue to introduce the server as THUNDERDOME with version 0.218.12.
I found also some logs in /home/user/.config/unity3d/IronGate/Valheim/ with the name THUNDERDOME in it, I've removed it and it continue to introduce the server as THUNDERDOME. There's something I missing but I found a solution. I even deleted all steam cloud files, install the game on different disk.

I'm running the game under Linux Arch, if somebody has an explaination to this issue I take it ^^.

Juste change the exposed port from 2456-2458 to 2556-2558 and no issue...
